osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPointConnection.cs

```diff
@@ -110,8 +110,11 @@ public static void GetFadeTimes(OsuHitObject start, OsuHitObject end, float frac
             double startTime = start.GetEndTime();
             double duration = end.StartTime - startTime;
 
+            // For now, adjust the pre-empt for approach rates > 10.
```

Comment:
Is this to match how it behaves in stable?

Reply:
Stable's also static at 800 at all times, with the only thing changing being the animation speed.

I did intend to make it match but after thinking about it again this is actually incorrect as it should be a constant 33% reduction rather than based on preempt time, both here and with the change in `OsuHitObject`.

Reply:
This likely deserves a better inline explanation I think. I'm still not really sure how this works, even with your comment above (and if I'm not sure, it's probably going to be equally hard for others to figure out).